params/precompile_config.go

```diff
@@ -129,7 +129,7 @@ func (c *ChainConfig) verifyPrecompileUpgrades() error {
 			disabled     bool
 		)
 		// check the genesis chain config for any enabled upgrade
-		if config, ok := c.PrecompileUpgrade.getByKey(key); ok {
+		if config, ok := c.PrecompileUpgrade.getByKey(key); ok && config.Timestamp() != nil {
```

Comment:
Should this return an error if there is a nil block timestamp rather than skipping?